docs/authorization-introduction.md

```diff
@@ -44,30 +38,35 @@ A policy looks like:
 ```
 
 The main elements of a policy are:
-- The policy ID,
-- The policy version,
-- The policy name,
+- The policy ID
+- The policy version
+- The policy name
 - The policy statements that contains an array of statements.
 
 The main elements of a Statement are:
-- The Action to be performed. Example: `'authorization:teams:create'` or `'authorization:organizations:*'`,
-- The Resource on which the action is performed. Example: `'FOO:orga:CLOUDCUCKOO:scenario:*:entity:north-america-id'`,
-- The Effect - has the value 'Allow' or the value 'Deny'.
+- The Action to be performed. Example: `'authorization:teams:create'` or `'authorization:organizations:*'`
+- The Resource on which the action is performed. A Resource name is effectively a URI for your resources. Example: `'FOO:orga:CLOUDCUCKOO:scenario:*:entity:north-america-id'`
+- The Effect - has the value 'Allow' or 'Deny'.
+
+Note that wildcards can be used in Action and Resource names, as can certain variables, see [IAM Policy Variables Overview]() for more details.
+
+For a detailed description of Policies, see the [AWS Policy Elements Reference]().
+
+## API
 
-Wildcards can be used in Action and Resource names.
+The Udaru API can be used directly as a normal node module, but also has a REST API (a Hapi plugin is provided), and it can also be used out of the box as a standalone server. 
 
-## The authorization system structure
+The API has a mix of public and private routes:
 
-The Udaru authorization system uses the same PBAC approach to perform administration of its internal resources as it does for controlling access to external resources. More details can be seen on [Authorization Technical Specs][]
+- Routes that validate if actions on resources are authorized are exposed as [public routes][]
+- [private routes][] are routes that can be accessed only through a **Service Key**. These private routes provide functionality to create, update and delete policies.
+- Routes to access and manage the Organizations, Teams, Users and Policies hierarchies are exposed as [public routes][].
 
-The exposed functionality endpoints are structured like the following:
-- Routes that validate if actions over resources are authorized are exposed as [Public routes][],
-- [Private routes][] are routes that can be accessed only through a **Service Key** that is passed as a query string in the form `sig=<key>`. The **Service Key** is loaded by Udaru from the config file. These private routes provide functionality to create, update and delete policies,
-- Routes to access and manage the Organizations, Teams, Users and Policies hierarchies are exposed as [Public routes][].
+### REST API - Hapijs plugin 
 
-## Routes in stand-alone server and Hapijs plugin modes
+TODO - this section needs work, or should we nuke this section? Seems a little complicated to belong in the introduction. 
```

Comment:
added that as a hint to the user on what's the connection between the routes and the policies attached to users to be able to access the endpoint. I could simplify it.